### PR TITLE
Add various manifests

### PR DESCRIPTION
**Pull request by yanc0**
Things added to the cluster:

- devtoolsv1
  - (Helm) Jenkins (as to stay in develop environment)
- appv1
  - (Helm) Redis
  - Helloworld

Helloworld does not use Redis (code is very simple) but the goal is not to get the best helloworld ever, but manage multiple services in multiple environments. So, I think it's OK for now.

Tested with Flux and Flux-Helm-Operator in GKE 1.10.

```
NAME                                  READY     STATUS    RESTARTS   AGE
appv1-helloworld-84b7885fbd-gdlvr     1/1       Running   0          10m
appv1-helloworld-84b7885fbd-xt7qk     1/1       Running   0          10m
appv1-redis-675cdd5c98-jlzv7          1/1       Running   0          30m
devtoolsv1-jenkins-5bfd768946-kc54x   1/1       Running   0          35m
flux-664d4788bc-ffglp                 1/1       Running   0          4h
flux-helm-operator-7b9676cd69-75jpt   1/1       Running   0          3h
memcached-6879d9d8d-8tcgn             1/1       Running   0          17h
```

### CODE REVIEW

File: charts/jenkins/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+approvers:
```
Comment (hiddeco) [inline charts/jenkins/OWNERS:1]: Obsolete, this is used by the upstream (https://github.com/kubernetes/charts) to define some permissions.

**Review by hiddeco — COMMENTED** [review]
Gave it a quick reading.

Think it would be smart to add labels to the manifests that reflect the environment.

File: charts/jenkins/OWNERS (hunk 1 above)
Comment (yanc0) [reply charts/jenkins/OWNERS:1]: It was added by running: `helm fetch --untar stable/jenkins ` in charts directory. Do we need to clean the upstream every time? I think the pro of Helm is to quickly get community charts and install them to your cluster. As they have no impact, cleaning them may be over kill, I think, no?

Comment (hiddeco) [reply charts/jenkins/OWNERS:1]: Valid point, reason why it works for the `redis` chart in that case is that it's included in their `.helmignore`. May be worth pushing a fix to the upstream.

**Comment by yanc0** [comment]
@hiddeco I added labels on manifests

File: charts/jenkins/OWNERS (hunk 1 above)
Comment (yanc0) [reply charts/jenkins/OWNERS:1]: I made a PR on the Jenkins chart. https://github.com/kubernetes/charts/pull/5821

**Review by hiddeco — APPROVED** [review]

**Pull request merged**